oci_runtime.rst

```diff
@@ -31,168 +31,203 @@ tools from the OCI ecosystem.
-Specification closely. Instead, it uses its own runtime that is better matched
-to the requirements and limitations of multi-user shared compute environments.
-From 3.11, the experimental ``--oci`` mode :ref:`can run containers using a true
-OCI runtime <oci_mode>`.
-
-**OCI Runtime CLI** - The ``singularity oci`` commands were added to provide a
-mode of operation in which {Singularity} does implement the OCI runtime
-specification and container lifecycle. These commands are primarily of interest
-to tooling that might use {Singularity} as a container runtime, rather than end
-users. End users will general use the ``--oci`` mode with ``run / shell /
-exec``.
-
-Future Development
-==================
-
-As newer Linux kernels and system software reach production environments, many
-of the limitations that required {Singularity} to operate quite differently from
-OCI runtimes are becoming less-applicable. From 3.11, {Singularity} development
-will focus strongly on greater OCI compliance for typical usage, while
-maintaining the same ease-of-use and application focus.
-
-You can read more about these plans in the following article and open community
-roadmap:
-
-* https://sylabs.io/2022/02/singularityce-4-0-and-beyond/
-* https://github.com/sylabs/singularityce-community
-
-.. _oci_mode:
+1. The :ref:`OCI Mode section <oci_mode>` of this page introduces the OCI
+   runtime (``--oci``), which runs OCI / Docker containers in their native
+   format.
+2. The :ref:`OCI Command Group section <oci_command>` of this page documents the
+   ``singularity oci`` commands, which provide a low-level means to run
+   {Singularity} SIF containers with a command line that matches other OCI
+   runtimes.
+3. The :ref:`Support for Docker <singularity-and-docker>` page discusses
+   limitations, compatibility options, and best practices for running OCI /
+   Docker containers with {Singularity}'s default runtime.
 
 ********************
 OCI Mode (``--oci``)
 ********************
 
-Overview
-========
-
-Beginning in {Singularity} 3.11, users can run an OCI / Docker container in its
-native format by adding the ``--oci`` flag to a ``run / shell /exec`` command:
+Users can run an OCI / Docker container in its native format by adding the
+``--oci`` flag to a ``run / shell /exec`` command:
 
 .. code::
 
-  $ singularity shell --oci docker://ubuntu 
+  $ singularity shell --oci docker://ubuntu
   2023/02/06 11:00:10  info unpack layer: sha256:677076032cca0a2362d25cf3660072e738d1b96fe860409a33ce901d695d7ee8
   Singularity> echo "Hello OCI World!"
   Hello OCI World!
 
 In ``--oci`` mode, the familiar ``singularity`` command line is used, and
-friendly defaults such as auto-mounting of the ``$HOME`` directory are still
-applied. The user experience is similar to when the ``--compat`` flag is used
-with the default runtime mode.
+friendly defaults familiar from the native {Singularity} runtime - such as
+auto-mounting of the ``$HOME`` directory - are still
```

Comment:
Oh, that's right... any thoughts on something else we could give here as an example of "friendly defaults familiar from the native SingularityCE runtime"? Or should we just omit the interjection altogether?

Reply:
On second thought, this will need a bit more explanation with regard to the `--no-compat` stuff coming in which does give you the native runtime behaviour. Let's tidy that up later.